src/12_scopes.py

```diff
@@ -19,6 +20,7 @@ def outer():
     y = 120
 
     def inner():
+        global y
```

Comment:
y is not defined in the global scope here. It is set to 120 inside the outer function. Try `nonlocal y` instead of `global y`